### PR TITLE
[0.9.x] Restore the version to 0.9.1-SNAPSHOT

### PR DESCRIPTION
**Pull request by menakaj**
### Purpose
This PR is to restore the version to 0.9.1-SNAPSHOT

### Issues

Fixes #

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [ ] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

File: resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml [hunk 1]
```diff
@@ -112,7 +112,7 @@ spec:
               name: router-keystore-vol
             - mountPath: /home/wso2/security/truststore
               name: router-truststore-vol
-          image: wso2/choreo-connect-router:0.9.0.2
+          image: wso2/choreo-connect-router:0.9.0
```
Comment (Rajith90) [inline resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml:115]: These versions should be snapshot

File: resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ spec:
               name: log4j2-vol
             - mountPath: /home/wso2/lib/dropins
               name: dropins-vol
-          image: wso2/choreo-connect-enforcer:0.9.0.2
+          image: wso2/choreo-connect-enforcer:0.9.0
```
Comment (Rajith90) [inline resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml:49]: These versions should be snapshot. Check all the places as well

Comment (menakaj) [reply resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml]: Fixed in https://github.com/wso2/product-microgateway/pull/2000/commits/24871c5ba5b2c1fba5acf2c97ef707f9fd591787

File: resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml (hunk 1 above)
Comment (menakaj) [reply resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml]: Fixed in https://github.com/wso2/product-microgateway/pull/2000/commits/24871c5ba5b2c1fba5acf2c97ef707f9fd591787

**Pull request closed without merging**